backends/ref/ceed-ref-restriction.c

```diff
@@ -431,6 +458,9 @@ static inline int CeedElemRestrictionApply_Ref_Core(CeedElemRestriction rstr, co
                                                                                elem_size, v_offset, uu, vv));
         }
         break;
+      case CEED_RESTRICTION_POINTS:
+        CeedCallBackend(CeedElemRestrictionApplyAtPointsInElement_Ref(rstr, num_comp, start, stop, t_mode, uu, vv));
```

Comment:
Why does this call `CeedElemRestrictionApplyAtPointsInElement_Ref` and not `CeedElemRestrictionApplyAtPoints_Ref`?

Reply:
I can adjust the name for clarity, but `CeedElemRestrictionApplyAtPointsInElement_Ref` accepts a range of elements to restrict to.

Reply:
I pushed some renaming that hopefully makes things a bit clearer

Reply:
Yep, those look good to me!